CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [9.125.0] - 2020-07-23
+
```

Comment:
Nice! ✨ 
Just a comment. Should this _Hide clear button on `InputSearch` when disabled`_ entry be on `### Fixed` section?

Reply:
Idk, I just rebase, it was already like that in PR.

Reply:
@vlaux what do you think?

Reply:
Nevermind... Already merged 🤷